docs/guides/authentication.md

```diff
@@ -48,6 +52,8 @@ export AZURE_CLIENT_SECRET="$(terraform output --raw azure_client_secret)"
 
 ## Google Cloud Platform
 
+[Create a GCP account](https://cloud.google.com/free) if needed, and then set the environment variable:
+
 - `GOOGLE_APPLICATION_CREDENTIALS` - Path to (or contents of) a service account JSON key file.
```

Comment:
```suggestion
Either one of:
- `GOOGLE_APPLICATION_CREDENTIALS` - Path to a service account JSON key file.
- `GOOGLE_APPLICATION_CREDENTIALS_DATA` - Contents of a service account JSON key file
```

Reply:
/CC @0x2b3bfa0 on this one :)

Reply:
```suggestion
Either one of:
- `GOOGLE_APPLICATION_CREDENTIALS` - **Path to** a service account JSON key file
- `GOOGLE_APPLICATION_CREDENTIALS_DATA` - **Contents of** a service account JSON key file
```

Reply:
🔔 @dacbd & @casperdcl, better now?

Reply:
iirc @0x2b3bfa0 we wanted to soft-deprecate `GOOGLE_APPLICATION_CREDENTIALS_DATA`? Not sure...

Reply:
I don't recall so. What we wanted to deprecate is `KUBERNETES_CONFIGURATION` in favor of the most consistent `KUBECONFIG_DATA` I think.

Reply:
Google Cloud and Kubernetes only take a path, and that's why we offer these unofficial alternative environment variables, so users don't have to expose secrets as files.